apps/dashboard/test/models/batch_connect/session_test.rb

```diff
@@ -504,4 +504,30 @@ def completed?
       assert_equal 1, base_dir.children.size
     end
   end
+
+  test 'default bc days old is set to 7' do
+    assert_equal 7, BatchConnect::Session.old_in_days
+  end
+
+  test 'can configure bc days old' do
+    with_modified_env({OOD_BC_CARD_TIME: '3'}) do
+      assert_equal 3, BatchConnect::Session.old_in_days
+    end
+  end
+
+  test 'return default if bc days old is less than or equal to 0' do
+    with_modified_env({OOD_BC_CARD_TIME: '-1'}) do
+      assert_equal 7, BatchConnect::Session.old_in_days
+    end
+
+    with_modified_env({OOD_BC_CARD_TIME: '0'}) do
+      assert_equal 7, BatchConnect::Session.old_in_days
+    end
```

Comment:
I think we do want to enable 0 days, that is a configuration that never keeps the cards. Sorry for all the back and forth! 

Will 0 work directly or are there errors when subtracting?